client/src/Component/Chart.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { scaleBand, scaleLinear } from 'd3-scale';
+
+import Axes from './Axes';
+import Bars from './Bars';
+import ResponsiveWrapper from './ResponsiveWrapper';
+
+class Chart extends Component {
+    constructor(props) {
+        super(props);
+        this.xScale = scaleBand();
+        this.yScale = scaleLinear();
+    }
+
+    render() {
+        const rawData = this.props.data;
+        // d.yr is number, while this.props.year is string.
+        const year = +this.props.year;
```

Comment:
Huh, I've never seen this syntax.  Does the + convert a string to a number?

Reply:
Yes! `+` convert a string to a number

Reply:
neat!